### PR TITLE
initial

### PR DESCRIPTION
**Pull request by acmk01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+#### Postgres setup DB
+```sql
+create role auth password 'pass' superuser login;
+create database auth owner auth;
+```
+
+#### DB migration
+```bash
+python auth/manage.py db migrate
```
Comment (matthieuchoplin) [inline README.md]: Trying to run this command gives me this error:
```
Traceback (most recent call last):
  File "/home/matt/PycharmProjects/auth-service/manage.py", line 2, in <module>
    from flask.ext.migrate import Migrate, MigrateCommand
  File "/root/venv-auth/lib/python3.4/site-packages/flask/exthook.py", line 81, in load_module
    reraise(exc_type, exc_value, tb.tb_next)
  File "/root/venv-auth/lib/python3.4/site-packages/flask/_compat.py", line 32, in reraise
    raise value.with_traceback(tb)
  File "/root/venv-auth/lib/python3.4/site-packages/Flask_Migrate-1.8.0-py3.4.egg/flask_migrate/__init__.py", line 5, in <module>
    from alembic import __version__ as __alembic_version__
ImportError: No module named 'alembic'
```
I installed SQLAlchemy (I think it is missing from the `requirements.txt`), but I am still getting the result.

Comment (matthieuchoplin) [inline README.md]: I think alembic is also missing from the dependencies

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+python auth/manage.py db upgrade
+```
+
+#### application start
+```bash
```
Comment (matthieuchoplin) [inline README.md]: you need to add a line after `bash` to have a proper rendering

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
-# auth-service
\ No newline at end of file
+## auth-service
+
+#### Development installation
+```bash
```
Comment (matthieuchoplin) [inline README.md]: add a line here for the rendering

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+git clone git@github.com:acmk01/auth-service.git
+cd auth-service
+git checkout <your dev branch>
+pip install -e .
+```
+#### Postgres setup DB
+```sql
```
Comment (matthieuchoplin) [inline README.md]: and here

File: README.md (hunk 2 above)
Comment (acmk01) [inline README.md]: will do. I knew it must be something like it

File: README.md (hunk 1 above)
Comment (acmk01) [inline README.md]: changed instructions

**Comment by matthieuchoplin** [comment]
:+1:

**Pull request merged**